### ISSUE TITLE
Slashes incorrectly escaped in output?

### ISSUE THREAD

**Issue by adrianbruntonsagecom** [43081j/postcss-lit#51]
We've noticed that slashes in the CSS output are doubling in quantity. Therefore, this:

```ts
  static override styles = css`
  .block {
    content: "\\e90e";
  }
  `;
```

Is becoming this:
```ts
  static override styles = css`
.block {
    content: "\\\\e90e";
  }
  `;
```

I've made a reproduction of the problem here:
https://github.com/adrianbruntonsagecom/postcss-lit-escaping-issue

To see it:
Clone repo:
`git clone https://github.com/adrianbruntonsagecom/postcss-lit-escaping-issue.git`

Install dependencies:
`npm i`

Run postcss:
`npm start`

You'll see output.ts is generated with twice the number of slashes in the content property than expected.

**Comment by 43081j** [comment]
i suspect i've fixed this in #52 FYI

**Comment by adrianbruntonsagecom** [comment]
> i suspect i've fixed this in #52 FYI

Looks like it does the job thanks :)
